### PR TITLE
Adding navigation and making the website a Single Page Application

### PR DESCRIPTION
**Pull request by BertrandConxy**
## Awesome Books: adding the navigation and making it SPA

In this branch, we have done the following:
- Modified the awesome books application to have a navigation bar.
- Modified the application to have three content sections: book list, add book form, and contact info.
- Used the wireframe as a guide to style the application.
- we created and implemented the class to make the application Single Page Application.
- We made sure that when you click on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -65,6 +65,29 @@ class DisplayBookList {
...
 
+// class to make the web app an SPA
+
+class DisplaySection {
+  constructor() {
+    this.pages = document.querySelectorAll('.page');
+  }
+
+  static DisplayOnly() {
+    // const page = document.querySelectorAll('.page')
```
Comment (GzimAsani) [inline app.js]: - [ ] Please remove the commented out code.

File: style.css [hunk 2]
```diff
@@ -1,3 +1,53 @@
+header {
+  display: flex;
```
Comment (GzimAsani) [inline style.css]: - [ ] You have used `display: flex` 7 times. Instead create a class with display: flex and use it whenever you need it

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @BertrandConxy and @butlermuwo                                     
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] Check the comments below :arrow_double_down: 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this 
You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @BertrandConxy👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes

- [ ] **Optional:** I highly recommend, adding a screenshot to your Readme for visitors to have a visual representation of your app.

_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by BertrandConxy** [comment]
Thank you @brytebee.We appreciate your review but your review is optional as you indicated. We are suggesting that you may change the status to approved because all the essential project requirements are met, then we will be adding the screenshot without needing to request another review.

**Review by brytebee — APPROVED** [review]
Hi Team👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BertrandConxy** [comment]
Thank you @brytebee

**Pull request merged**